index.js

```diff
@@ -33,9 +36,24 @@ module.exports = async function (
       continue;
     }
 
+    if (onlyOwner && actions[i].owner !== onlyOwner ){
+      debug("skipping owner:",actions[i].owner)
+      continue;
+    }
+
+    if (onlyRepo && onlyRepo !== "*" && actions[i].repo !== onlyRepo ){
```

Comment:
Rather than checking `*` explicitly, use `matcher` like the [existing functionality](https://github.com/mheap/pin-github-action/blob/main/checkAllowedRepos.js#L18)